rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,193 @@
+      example: { "library_name" : "mscoree.dll", "imported_functions" : "GetFileVersionInfoSizeA" }
+
+    - name: sections
+      level: extended
+      description: >
+        Data about sections of compiled binary PE
+      type: keyword
+      example: { "chi2" : 3027194, "virtual_address": 8192, "entropy": 6.24, "flags": "rx", "name": ".text", "raw_size": 198144 }
+
+    - name: resource_details
+      level: extended
+      type: keyword
+      description: >
+        If the PE contains resources, some info about them
+      example: { "chi2": -1, "filetype": "English text", "entropy": 0, "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855", "lang": "CHINESE SIMPLIFIED" }
+
+    - name: resource_languages
+      level: extended
+      type: keyword
+      short: List of resource languages.
+      description: >
+        Digest of languages found in resources. Key is language (as string) and
+        value is how many resources there are having that language (as integer)
+      example: { "ENGLISH US": 1, "CHINESE SIMPLIFIED": 760 }
+
+    - name: resource_types
+      level: extended
+      type: keyword
+      short: List of resource types.
+      description: >
+        Digest of resource types. Key is resource type (as string) and value
+        is how many resources there are of that specific type (as integer)
+      example: { "RT_VERSION": 1, "RT_MANIFEST": 1 }
+
+    - name: packers
+      level: extended
+      type: keyword
+      short: Identifies packers used.
+      description: >
+        Identifies packers used on Windows PE files by several tools and AVs.
+        Keys are tool names and values are identified packers, both strings.
+        see `file.pe.packers` for merged list of packers from all tools.
+      example: { "tool_name": "PEiD", "name": ".NET executable" }
+
+    - name: exports
+      level: extended
+      type: keyword
+      description: >
+        List of symbols exported by PE
+      example: DllInstall, DllRegisterServer, DllUnregisterServer
+
+    - name: creation_date
+      level: extended
+      short: Build or compile date.
+      description: >
+        Extracted when possible from the file's metadata. Indicates when it was
+        built or compiled. It can also be faked by malware creators.
+      type: date
+      example: "2020-11-05T17:25:47.000Z"
+
+    - name: authentihash
+      level: extended
+      description: >
+        Authentihash of the PE file.
+      type: keyword
+      example: ac9555d914bbb112ecc5f15bb9887ca8371f493ab0941344e976bb8410c8aa78
+
+    - name: compile_timestamp
+      level: extended
+      description: >
+        Compile timestamp of the PE file.
+      type: date
+      example: "2020-11-05T17:25:47.000Z"
+
+    - name: compiler_product_versions
+      level: extended
+      type: keyword
+      description: >
+        Version of the compiler.
+      example: VS98 (6.0) build 8168
+
+    - name: rich_pe_header_hash
```

Comment:
Since rich headers are structured (even though undocumented) and can contain compiler-level information. Does it make sense to move this into its own set of fields (i.e. make this `pe.rich_header.hash.md5`)?